ctags/parsers/geany_pascal.c

```diff
@@ -51,7 +51,7 @@ static void createPascalTag (tagEntryInfo* const tag,
 	else
 	{
 		/* TODO: Passing NULL as name makes an assertion behind initTagEntry failure */
```

Comment:
Not so sure about this comment but I left it as it is also still present in the uctags Pascal parser.